spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/config/AdminConfiguration.java

```diff
@@ -89,64 +96,77 @@
 	private String dataSourceUrl;
 
 	@Bean
+	@ConditionalOnMissingBean
```

Comment:
Given there will be a single `AdminConfiguration` in the context, what is the significance of using `ConditionalOnMissingBean` here?